config.yml

```diff
@@ -381,6 +385,17 @@ steps:
 
           Like before, now we have to wait for the workflow to run so that we can see the results.  
 
+          <details>
+            <summary>Workflow not running? Click here for some troubleshooting.</summary>
+
+          Try the following troubleshooting steps:
+          1. Click on the [Actions tab]({{ store.actionsUrl }}) to see the status of your workflow run. See [Managing a workflow run](https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run) on GitHub Help for more information.
+          1. Edit your [workflow file]( {{ store.workflowEditUrl }}) and look for errors in the linter built into the browser.
+          1. Look for the [workflow trigger](https://help.github.com/en/actions/reference/events-that-trigger-workflows) and ensure you are performing an action that triggers that workflow.
```

Comment:
> however the workflow still doesn't run because it is triggered by the opening of an issue. so i had to create another issue

It makes sense now, thank you @lmkeston! Does this troubleshooting step not capture that?

Reply:
it does, but i'm not sure learners would realize that the issue that was just added won't trigger it after fixing the code, that a new issue is required to unblock.  your call @hectorsector!

Reply:
What do you think @mattdavis0351?

Reply:
@hectorsector it seems as though it captures it in the instructions, and both of you agree on that.  Let's run it like this to do some testing.  If we see this come up often in the forum, then we can adjust the wording.

I'd hate to let a hypothetical understanding of instructions derail shipping this.  Let' ship to learn 🤷‍♂️

Reply:
Yay! Unsolicited feedback from me, let's ship to learn!

![](https://media.giphy.com/media/l09y7RIxj7X12S24gN/giphy.gif)